src/snippets/bot-protection/quick-start/bun/Step3.js

```diff
@@ -27,6 +27,10 @@ export default {
       return new Response("Forbidden", { status: 403 });
     }
 
+    if (decision.reason.isBot() && decision.reason.isSpoofed()) {
```

Comment:
```suggestion
    if (decision.reason.isBot() || decision.reason.isSpoofed()) {
```

Shouldn't all these be OR? That's how you used it in [the SDK repo examples](https://github.com/arcjet/arcjet-js/pull/2375/files#diff-d87647aacb2ee3bb9ef0c144004d6db6d13443b0aa0ff79fdbb83e3474a263a3R37).

Reply:
This needs to be `&&` because the type needs to narrow on `isBot()`. The example you linked has `isBot()` as a separate `if` statement wrapping the section that checks `isSpoofed()` because it is "denied or spoofed"